index.html

```diff
@@ -355,6 +359,14 @@ <h2><code>PointerEvent</code> Interface</h2>
                         <dd>
                             <p>The clockwise rotation (in degrees, in the range of [0,359]) of a transducer (e.g. pen stylus) around its own major axis. For hardware and platforms that do not report twist, the value MUST be 0.</p>
                         </dd>
+                    <dt><dfn>altitudeAngle</dfn></dt>
+                        <dd>
+                            <p>The altitude (in radians) of the transducer (e.g. pen stylus), in the range [0,π/2] - where 0 is parallel to the X-Z plane, and π/2 is perpendicular to the X-Z plane. The value 0 should be used for devices which do not support this property.</p>
+                        </dd>
+                    <dt><dfn>azimuthAngle</dfn></dt>
+                        <dd>
+                            <p>The azimuth angle (in radians) of the transducer (e.g. pen stylus), in the range [0, 2π] - where 0 represents a transducer whose cap is pointing in the direction of increasing X values on the X-Z plane, and π/2 represents a transducer whose cap is pointing in the direction of increasing Z values on the X-Z plane. The value 0 should be used for devices which do not support this property.</p>
```

Comment:
Based on https://w3c.github.io/touch-events/#dom-touch-azimuthangle, same comment as for altitudeAngle, I believe it should be plane X-Y not X-Z. As well, I feel for azimuth angle π/2 the axis should be Y, not Z. The definition of azimuth angle for touch events seems to me more intuitive.

Reply:
Agree, the X-Z thing was my mistake. I've expanded this (also adding a clock analogy, which may be easier to convey/understand) and the fact that the value increases when going clockwise